website/src/pages/ceros-full-page-takeover.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FunctionComponent } from 'react'
+import { PageProps } from 'gatsby'
+import { Helmet } from 'react-helmet'
+
+import Layout from '../components/Layout'
+
+const CerosFullPageTakeover: FunctionComponent<PageProps> = props => (
+    <Layout location={props.location}>
+        <Helmet>
+            <script
+                type="text/javascript"
+                src="https://view.ceros.com/scroll-proxy.min.js"
+                data-ceros-origin-domains="https://view.ceros.com/"
+            />
+        </Helmet>
```

Comment:
This might be a good time to create a CerosScript component that you can import and use in Ceros pages so you can reuse the same script and we have one point of reference.

We could eventually even make `CerosPage` or `CerosContent` components depending on what the props are.

Reply:
I like that idea! This branch isn't being merged in though, since it's just to test our trial with Ceros. I think we should hold off on optimizing the code until we incorporate it more permanently. That sound okay?

Reply:
Yes, definitely, totally agree!